components/collector/src/source_collectors/performancetest_runner/performancetest_scalability.py

```diff
@@ -15,7 +15,7 @@ async def _parse_value(self, responses: SourceResponses) -> Value:
         return str(min(breaking_points))
 
     async def _parse_total(self, responses: SourceResponses) -> Value:
-        """Override to compute the total number of vusers from the responses."""
+        """Override to compute the total number of users from the responses."""
```

Comment:
"vusers" is actually correct here, it's short for "virtual users"

```suggestion
        """Override to compute the total number of virtual users from the responses."""
```